### ISSUE TITLE
Kategória felvitel a feladat látrehozás oldalon

### ISSUE THREAD

**Issue by maxinteger** [zsebtanar/zsebtanar-proto#41]
Nem létező elem (osztály, tantárgy, témakör, címke ...) esetén autómatikusan mentjük az új nevet.

**Comment by zsebtanar** [comment]
@maxinteger itt sztem lehet egy probléma: az, hogy ha két tantárgyat is kiválasztasz, akkor hova mentse el a témakört? sztem az a legjobb, ha a tantárgyhoz csak 1 dolgot lehessen választani (ez akár egy legördülő menü is lehet). és így akkkor egyértelmű, hogy hova mentse.

**Comment by maxinteger** [comment]
@zsebtanar Szerintem ez nem fog problémát okozni, mert minden megjelölt tantárgyhoz külön témakör választó jelenik meg, de ettől függetlenűl korlátozhatjuk a megjelölehtő tárgyak számát.

**Comment by zsebtanar** [comment]
@maxinteger igen, most ez még jó, de ha a feladatszerkesztőnél is meg lehet adni témakört egyből, akkor ez több kiválasztott tantárgynál gondot okozhat, nem?
![image](https://user-images.githubusercontent.com/13850825/28521381-af627efc-7073-11e7-8536-c3eba82aa40f.png)

**Comment by maxinteger** [comment]
@zsebtanar A feladat szerkesztő egy teljesen külön történet, az is lehet, hogy ott nem is lesz tantárgy/témakör mező VAGY a berakott feladatokból vesszük ki ezeket az értékeket VAGY kitalálhatunk valami teljesen mást. Egyenlőre nem teljesen értem a problémát, de ha nem most még nem jelentkezik az átalad leírt hiba akkor hagyjunk mindent így és térjünk vissza rá ha valóban gondot okoz.

**Comment by zsebtanar** [comment]
@maxinteger mindenképp jó lenne, ha egy új témakört ott helyben létre tudnék hozni a feladat szerkesztése közben, és nem kellene kilépnem, megnyitnom a témakörök listáját, és ott bővíteni a listát. én azt javaslom, hogy egy feladatnál csak 1 tantárgyhoz lehessen kiválasztani. így egyértelmű, hogy ha új témakört adok meg, az melyik tantárgyhoz tartozik.

**Comment by maxinteger** [comment]
@zsebtanar ne haragudj de továbbra sem értem mi a prboléma a több tantárgy megjelölésével. Minden tágy külön témakör választót kap és amikor felveszel egy új témakört akkor az ahhoz a tárgzhoz veszed fel amelzik tárgy témakörével dolgozól, tehát nem tudsz felvenni matek témakört a "Fizika témakörök" választóból és fordítva sem. Szerintem ez így elég egyértelmű, de ha tudsz olyan esetet amikor nem az szólj.
Ettől függetlenűl lekorlátozhatom a választható tárgyak számát, de egzébként sem kötelező, hogy többet megjelölj ;)

**Comment by zsebtanar** [comment]
@maxinteger megpróbálok mondani egy példát:
- létrehozol egy feladatot
- a feladatnál kiválasztasz két tárgyat (pl. matek + fizika)
- tegyük fel, hogy a megadott témakörök között nem találod azt, ahova a feladat tartozik, úgyhogy hozzá szeretnél adni egy új témakört (jelen pillanatban ehhez külön oldalra kell menned, de a jövőben jó lenne, ha ez kényelmesebb lenne: csak beírod a feladat szerkesztői oldalon az új témakört, nyomsz egy entert, és automatikusan hozzáadja az új témakört)
- és itt jön a probléma: melyik tantárgyhoz adja hozzá a rendszer az új témakört? mindkettőhöz?
remélem, így már picit érthetőbb...
